src/main/java/eu/deltasource/exception/AccountTypeCannotBeDifferentFromCurrentAndSavingsException.java

```
@@ -0,0 +1,8 @@
+package eu.deltasource.exception;
+
+public class AccountTypeCannotBeDifferentFromCurrentAndSavingsException extends RuntimeException{
```

Not really needed - You can throw this exception instead - InvalidInputException.